CHANGELOG.md

```diff
@@ -0,0 +1,24 @@
+# 1.0.0 - 2017-02-12
+
+**BREAKING CHANGES**
+- `createCycleMiddleware()` no longer takes any arguments. Instead you need to call `Cycle.run` yourself passing it your `main` function and `drivers` explicitly:
```

Comment:
Is it also worth mentioning here and in the readme the need to `[npm i -s/yarn add] xstream-run`?